web-ui/view/tyxml_helpers.ml

```diff
@@ -0,0 +1,8 @@
+let x_cloak = Tyxml.Html.Unsafe.string_attrib "x-cloak" ""
```

Comment:
We should add the recurrent Tyxml structure we use to simplify our code flow. For example,
`a ~a:[href = ""] [txt ...]` or others that are used often. WDYT?

Reply:
I wasn't sure if this might lead to a bit of confusion for readability (since the template will consist of standard HTML but also these simplifications) but on reflection I think this is a good idea - I'd like to introduce this in a consolidation step in the next PR perhaps.